### PR TITLE
V1.14.2 release

### PR DESCRIPTION
**Pull request by adnxn**
* Bug - Added checks for circular task dependencies. #796
* Bug - Fixed an issue with Docker auth configuration overrides. #751
* Bug - Fixed a race condition in the task clean up code path. #737
* Bug - Upgrade AWS SDK to v1.8.19 to include SDK bugfixes. #786
* Bug - Fixed an issue involving concurrent map writes. #743

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 1.14.2
+* Bug - Added checks for circular task dependencies. [#796](https://github.com/aws/amazon-ecs-agent/pull/796)
+* Bug - Fixed an issue with Docker auth configuration overrides [#751](https://github.com/aws/amazon-ecs-agent/pull/751)
+* Bug - Fixed a race condition in the task clean up code path. [#737](https://github.com/aws/amazon-ecs-agent/pull/737)
+* Bug - Upgrade AWS SDK to v1.8.19 to include SDK bugfixes [#786](https://github.com/aws/amazon-ecs-agent/pull/786)
```
Comment (aaithal) [inline CHANGELOG.md]: I don't think this is a bug. In fact, I don't think we should call it out at all as it's not a customer facing thing.

Comment (adnxn) [inline CHANGELOG.md]: @aaithal which one exactly?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
...
+* Bug - Fixed an issue involving concurrent map writes [#743](https://github.com/aws/amazon-ecs-agent/pull/743)
+
```
Comment (aaithal) [inline CHANGELOG.md]: Also, you're missing the `Enhancement` to introspection API for querying task id's with short container ids #813

File: CHANGELOG.md (hunk 1 above)
Comment (aaithal) [inline CHANGELOG.md]: `Bug - Upgrade AWS SDK to v1.8.19 to include SDK bugfixes [#786]`

Comment (adnxn) [inline CHANGELOG.md]: gotcha. removed.

File: CHANGELOG.md (hunk 2 above)
Comment (adnxn) [inline CHANGELOG.md]: added.

**Comment by vsiddharth** [comment]
LGTM

**Pull request merged**